### DISCUSSION TITLE
git checkout V.10.0.0 , switch this tags。run the command line :docker-compose -up -d happed error

### DISCUSSION THREAD

**Discussion by Leelongqi** [getredash/redash#6479] [Q&A] [answered]
![image](https://github.com/getredash/redash/assets/33893308/677465d1-2c82-46c0-8736-ca22a38a6cfb)

**Comment by justinclift** [comment]
This one's a bit harder to really give advice for.  The v10 series official releases were designed for a build process prior to the "Community Led" relaunch of Redash.

So, the `make build` and `make compose_build` steps *might* not be quite right for those older tags.  I'm not sure personally.

**Reply by Leelongqi** [reply]
thanks,why don‘t have trino datasource after build redash sever?,redash version:23.09.0-dev (latest master )https://github.com/getredash/redash/discussions/6477

**Reply by Leelongqi** [reply, marked as answer]
redash:v10 have a python in-build package conflict ,need to lower the library version by changing to importlib-metadata<5.0
![image](https://github.com/getredash/redash/assets/33893308/9d8904dc-94e8-4f7b-8ab7-3c4cda690bd7)
